### PR TITLE
Convert to fact hash command and remove fact decrypt. Closes #251.

### LINKED ISSUE #251: Add rho fact hash and remove fact encrypt,  decrypt, and redact
## Specify type:
- Enhancement
___

## Description:
Similar function to  #220 however, instead of encrypting the data it will be hashed

### PR DESCRIPTION
**Pull request by chambridge**

### CODE REVIEW

File: rho/facthashcommand.py [hunk 1]
```diff
@@ -7,33 +7,41 @@
...
+import hashlib
 from rho import utilities
 from rho.clicommand import CliCommand
 from rho.translation import _
 from rho.utilities import multi_arg, _read_in_file, write_csv_data
-from rho.vault import get_vault
 
-PROMPT = "Please enter your encryption password: "
+
+def compute_md5_hash(input_string):
```
Comment (noahl) [inline rho/facthashcommand.py]: MD5 is not a cryptographically secure hash function any more. Not that this is an incredibly high-security application, but let's use SHA-256 (or something similar), which is guaranteed to be in `hashlib`.

**Review by noahl — APPROVED** [review]
One comment about the hash algorithm, otherwise LGTM

**Review by elyezer — APPROVED** [review]
I agree with @noahl comment about the hash func to be used. Thanks for the PR

File: rho/facthashcommand.py (hunk 1 above)
Comment (chambridge) [reply rho/facthashcommand.py]: @mdvickst Can you weigh in on this? You had specifically mentioned MD5 so I want to implement the correct thing.

Comment (mdvickst) [reply rho/facthashcommand.py]: I'm fine with SHA-256.

**Pull request merged**